collections-project/src/lesson14/AttendanceLog.java

```diff
@@ -0,0 +1,27 @@
+package lesson14;
+
+public class AttendanceLog {
+    private String user_id;
```

Comment:
userId should be better, just to avoid mismatch of code style. But it can be displayed as user_id